bin/runtime/src/lib.rs

```diff
@@ -648,10 +648,10 @@ parameter_types! {
     // Maximum size of the lazy deletion queue of terminated contracts.
     // The weight needed for decoding the queue should be less or equal than a tenth
     // of the overall weight dedicated to the lazy deletion.
-    pub DeletionQueueDepth: u32 = ((DeletionWeightLimit::get() / (
+    pub DeletionQueueDepth: u32 = (DeletionWeightLimit::get().saturating_div((
             <Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(1) -
             <Runtime as pallet_contracts::Config>::WeightInfo::on_initialize_per_queue_item(0)
-        )) / 10) as u32; // 2228
+        ).ref_time())).ref_time() as u32 / 10; // 2228
```

Comment:
nit: push `10` to saturing div and cast it to u32 at the end

Reply:
done